src/debug.ts

```diff
@@ -0,0 +1,217 @@
+let debugLayerFront: pc.Layer = null;
+let debugLayerBack: pc.Layer = null;
+
+class DebugLines {
+
+    app: pc.Application;
+
```

Comment:
not loving the empty lines here and below. is there a particular reason for them? could they be removed?

Reply:
This comes from the following linting rule:
https://eslint.org/docs/rules/lines-between-class-members

It mentions we can turn off the rule for single line class members (`"exceptAfterSingleLine": true`), perhaps thats the route forward. Keeping it enforced for functions.

Reply:
I think I'd prefer that. Which would you prefer though, lines or no lines?

Reply:
I'm for no lines after single line members too, it keeps the format of these typings consistent with their assignment in the constructor.